doc/stdlib/file.rst

```diff
@@ -412,6 +412,9 @@
 
    Convert a path to an absolute path by adding the current directory if necessary.
 
+   .. note::
+       The function calls ``abspath(p1, p2...)`` and ``abspath(joinpath(p1, p2...))`` are equivalent.
+
```

Comment:
@yuyichao: My bad, I had commented asking the same (whether docstrings need to be changed in source or in the .rst files in the original issue). Didn't receive a reply so went ahead. Does that mean, I have to change the docstrings in both the ```.rst``` files and the source.
Thanks.